### PR TITLE
P1 b1 r20 Paris

### PR DESCRIPTION
**Pull request by GrantSchreib**

### CODE REVIEW

File: bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+              <lb ed="#B"/>sed ab illo ne <unclear>substhas</unclear> manum 
+              <lb ed="#B"/>tuam</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e198">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de5520-Rd1e209">Tho<!--line jumps up-->bie historia capitulo 4</ref> /<quote xml:id="bHY6yh-de5520-Qd1e202">Ex substan<!--line jumps down-->tua fac elemosina et noli 
+              <lb ed="#B"/>auertere faciem tuam ab ullo pau<lb ed="#B"/>pere
+                ita enim fiet ut nec <unclear>aceauer<lb ed="#B"/>tetur</unclear> 
+            <pb ed="#B" n="10-r"/>
+            <cb ed="#B" n="a"/>
+            facies domini</quote>
```
Comment (jeffreycwitt) [inline bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml:106]: missing a line break here.

File: bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            <p xml:id="bHY6yh-de5520-d1e280">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de5520-Rd1e303">IEsus filius syrac in ecclesiastico capitulo 3</ref> /<quote xml:id="bHY6yh-de5520-Qd1e284">ele mo<lb ed="#B"/>sina resistit peccatis</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e290">
+              ¶ <ref xml:id="bHY6yh-de5520-Rd1e313">capitulo 4</ref> <quote xml:id="bHY6yh-de5520-Qd1e292">Elemo<lb ed="#B"/>sinam 
+              pauperis ne defraudes et oculos 
+              <lb ed="#B"/>tuos ne transuertas a paupere</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e300">
+                /<quote xml:id="bHY6yh-de5520-Qd1e302">Cor
```
Comment (jeffreycwitt) [inline bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml:154]: try to leave a space after the slash
```suggestion
                / <quote xml:id="bHY6yh-de5520-Qd1e302">Cor 
```

File: bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+              <lb ed="#B"/>¶ <quote xml:id="bHY6yh-de5520-Qd1e312">Rogationem <unclear>cum rvnlatione</unclear> ne ab<lb ed="#B"/>icias 
+              et ne auertas faciem tuam ab 
+              <lb ed="#B"/>egeno</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e318">
+              ¶ <ref xml:id="bHY6yh-de5520-Rd1e343">capitulo 14</ref> <quote xml:id="bHY6yh-de5520-Qd1e320">Secundum vires tuas 
+              <lb ed="#B"/>porrigens da pauperi</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e326">
+              /<ref xml:id="bHY6yh-de5520-Rd1e353">17 capitulo</ref> <quote xml:id="bHY6yh-de5520-Qd1e328">Ele<lb ed="#B"/>mosina
```
Comment (jeffreycwitt) [inline bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml:168]: space after the slash
```suggestion
              / <ref xml:id="bHY6yh-de5520-Rd1e353">17 capitulo</ref> <quote xml:id="bHY6yh-de5520-Qd1e328">Ele<lb ed="#B"/>mosina 
```

File: bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+              <lb ed="#B"/>canere ante te sicud ypocrite faciunt 
+              <lb ed="#B"/>in synagogis eorum</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e347">
+              ¶ <ref xml:id="bHY6yh-de5520-Rd1e378">capitulo 15</ref> <quote xml:id="bHY6yh-de5520-Qd1e349">Non est bo<lb ed="#B"/>num 
+              sumere panem sumere et da 
+              <lb ed="#B"/>filiorum et dare canibus</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e355">
+              /<ref xml:id="bHY6yh-de5520-Rd1e390">captiulo 19</ref> <quote xml:id="bHY6yh-de5520-Qd1e359">Aut
```
Comment (jeffreycwitt) [inline bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml:183]: spacing
```suggestion
              / <ref xml:id="bHY6yh-de5520-Rd1e390">captiulo 19</ref> <quote xml:id="bHY6yh-de5520-Qd1e359">Aut 
```

File: bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+              <lb ed="#B"/>ussoldatur <unclear>tu nero</unclear> crucilitis et in
+              <lb ed="#B"/>hiis <unclear>onunly</unclear> vitus nos et uos chaos
+              <lb ed="#B"/>magnum firmatum est vt <unclear>lrius</unclear>
+              <lb ed="#B"/>volunt huic cursite aduos <subst><add place="margin-right">non</add></subst> possunt
+              <lb ed="#B"/>nequam inde nec huc transmeare
+              <lb ed="#B"/>et Aut Rogo ergo <unclear>prince</unclear> ut mittas
+              <lb ed="#B"/>eum in dominum patris mei / habeo enim
+            <pb ed="#B" n="10-v"/>
+            <cb ed="#B" n="a"/>
+              <lb ed="#B"/>quinque fratres vt testetur illis ne
```
Comment (jeffreycwitt) [inline bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml:241]: ideally the `cb` and `pb` element should be indented flush with the `lb` elements.
```suggestion
              <lb ed="#B"/>eum in dominum patris mei / habeo enim
              <pb ed="#B" n="10-v"/>
              <cb ed="#B" n="a"/>
              <lb ed="#B"/>quinque fratres vt testetur illis ne
```

File: bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+            <pb ed="#B" n="9-v"/>
+            <cb ed="#B" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de5520">
+            <head xml:id="bHY6yh-de5520-Hd1e108">Rubrica 20</head>
+            <head xml:id="bHY6yh-de5520-Hd1e111" type="question-title">De elemosyna</head>
+            <p xml:id="bHY6yh-de5520-d1e114">   
+              De elemosina Rubrica 20
```
Comment (jeffreycwitt) [inline bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml:64]: double check to make sure there isn't a missing line break here at the beginning.

File: bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml [hunk 7]
```diff
@@ -0,0 +1,302 @@
...
+              <ref xml:id="bHY6yh-de5520-Rd1e116">Psalmista pslamus 40</ref> <quote xml:id="bHY6yh-de5520-Qd1e123">Beatus quae 
+              <lb ed="#B"/>intelligit super egenum et pau<lb ed="#B"/>perem 
+              indie mala liberabit eum 
+              <lb ed="#B"/>dominus</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e133">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de5520-Rd1e132">Salo<!--line jumps up-->mon prouerb capitulo 14</ref> <quote xml:id="bHY6yh-de5520-Qd1e137">Qui mi<!--line jumps down-->seretur pauperi beatus erit</quote>
+            </p>
+            <p xml:id="bHY6yh-de5520-d1e144">  
+              <lb ed="#B"/>¶ <ref xml:id="bHY6yh-de5520-Rd1e143">17 capitulo</ref> <quote xml:id="bHY6yh-de5520-Qd1e146">Qui despicit pauperem ex<lb ed="#B"/>probrat
```
Comment (jeffreycwitt) [inline bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml:76]: Looks like we are missing the "break='no'" attribute throughout.

```suggestion
              <lb ed="#B"/>¶ <ref xml:id="bHY6yh-de5520-Rd1e143">17 capitulo</ref> <quote xml:id="bHY6yh-de5520-Qd1e146">Qui despicit pauperem ex<lb ed="#B" break="no"/>probrat 
```

File: bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml (hunk 1 above)
Comment (GrantSchreib) [reply bHY6yh-de5520/cod-bxxqqq_bHY6yh-de5520.xml]: corrected

**Comment by GrantSchreib** [comment]
all corrections made

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**